src/composables/useHotKey/index.ts

```diff
@@ -134,7 +136,15 @@ export function useHotKey(
 		if (options.caseSensitive) {
 			return event.key === key
 		}
+
+		if (latinLettersRegex.test(key) && nonLatinLettersRegex.test(event.key)) {
+			// If received key is a single letter, but non-Latin, derive it from key code
+			const parsedKey = event.code.replace('Key', '').toLowerCase()
+			return parsedKey === key.toLowerCase()
+		}
+
 		return event.key.toLowerCase() === key.toLowerCase()
+			|| event.code === key
```

Comment:
Imho if we fix it with your current solution we should revert all of the "allow passing `code`" thing.
It opens the gates for more issues :sweat_smile: